wavepool/views.py

```diff
@@ -14,9 +14,20 @@ def front_page(request):
             archive: the rest of the newsposts, sorted by most recent
     """
     template = loader.get_template('wavepool/frontpage.html')
-    cover_story = NewsPost.objects.all().order_by('?').first()
-    top_stories = NewsPost.objects.all().order_by('?')[:3]
-    other_stories = NewsPost.objects.all().order_by('?')
+    cover_story = None
```

Comment:
See comments in admin.py, but I think it is safe to assume that there will be exactly one cover story in the database. If we make that assumption, then we can simply do
```cover_story = NewsPost.objects.get(is_cover_story=True)```

If either no cover stories, or multiple cover stories are valid states, then this line should be something more along
the lines of 
```cover_story = NewsPost.objects.filter(is_cover_story=True).order_by("-publish_date").first()```
which would cover both additional cases.
```